WebRoot/css/zm.css

```diff
@@ -2176,7 +2168,42 @@ div.CompDragArea * {
 }
 
 .ZmTaskEditView {
-	@PageBg@
+	@TaskEditView@
+}
+
+.ZmTaskEditView .ZmTaskEditHeader {
+	@TaskEditHeader@
+}
+
+.ZmTaskEditView .ZmTaskEditContent {
+	@TaskEditContent@
+}
+
+.ZmTaskEditHeader .ZmTaskEditHeaderTable {
+	@TaskEditHeaderTable@
+}
+
+.ZmTaskEditHeader .ZmTaskEditHeaderTable > tbody > tr > td {
+	@TaskEditHeaderTableCell@
+}
+
+.ZmTaskEditHeaderTable .ZmFieldsetContent td,
+.ZmTaskEditHeaderTable .ZmFieldsetContent div,
+.ZmTaskEditHeaderTable .ZmFieldsetContent label {
+	@TaskEditViewFontSize@
```

Comment:
can't we just set font size for .ZmTaskEditHeaderTable .ZmFieldsetContent? 
Can we have some container defining property name in case someone want's to add more css rule to same selectors not related to font size?